src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-mimesis.py

```diff
@@ -0,0 +1,15 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2020 PyInstaller Development Team.
```

Comment:
2022 (even though it does sometimes feel like it's still 2020...).

Reply:
Done